CMSIS-OS/ChibiOS/ST_NUCLEO64_F411RE_NF/nanoCLR/halconf.h

```diff
@@ -130,7 +130,7 @@
  * @brief   Enables the SERIAL subsystem.
  */
 #if !defined(HAL_USE_SERIAL) || defined(__DOXYGEN__)
-#define HAL_USE_SERIAL              TRUE
+#define HAL_USE_SERIAL              FALSE
 #endif
```

Comment:
Here too..

Reply:
I've checked the board of course, and except the 80004005 debug error it works. Deploy goes fine, events raised for the switch.

Reply:
This is as it is on the ref boards too, so why would I use serial here?

Reply:
That's OK. as it was TRUE before I thought it could be a mistake.
Also was under the impression that this board was using the serial for Wire Protocol. 
If it's working it's working! 👍